docs/basic-docs/atomicdex-api-20-dev/withdraw_init.md

```diff
@@ -0,0 +1,63 @@
+# withdraw_init
+
+The `withdraw_init` method generates, signs, and returns a transaction that transfers the `amount` of `coin` to the address indicated in the `to` argument. It is only used for z coins like ZOMBIE, which may take some time to complete. The status of this method can be queried via the [withdraw_status](withdraw_status.html) method, or cancelled with [withdraw_status](withdraw_cancel.html).
```

Comment:
1) typo `cancelled with [withdraw_status](withdraw_cancel.html).` - I think this should be `cancelled with [withdraw_cancel](withdraw_cancel.html).` 

2) Could you tell me am I right or wrong? Right now links for `withdraw_status` and  `withdraw_cancel` are invalid. They lead to  our github project pages. But when this branch will be merged into master, links with `withdraw_status.html` and `withdraw_cancel.html` will exist, so everything will be fine.

3) `It is only used for z coins like ZOMBIE` - as I can see in our code the method `task::withdraw::init` is used [not only for `ZCoin`](https://github.com/KomodoPlatform/atomicDEX-API/blob/mm2.1/mm2src/coins/rpc_command/init_withdraw.rs#L115), but also for `UtxoCoin`, `QtumCoin`. In the case of other coins there will be an error `CoinDoesntSupportInitWithdraw`

Reply:
Thanks - I've reflowed these docs into two separate docs with subsections, and the sidebar accordingly
![image](https://user-images.githubusercontent.com/35845239/211017003-e61bfe36-a0dd-4f6c-863b-adee8c920c1d.png)
![image](https://user-images.githubusercontent.com/35845239/211017069-ba6b645e-223d-46de-bd41-b4c0af35a9f2.png)
Any z coin specific text was removed from the withdraw tasks doc, with the z coin doc noting that the withdraw tasks are required for those coins and a link to the withdraw task doc.

The links will work once deployed (you can [build & test locally](https://github.com/KomodoPlatform/developer-docs#setup) to confirm. They don't  work in github because the md / html extension change when built.